### ISSUE TITLE
Pulse gate circuit returned by ibmq_compile has empty calibrations

### ISSUE THREAD

**Issue by singular-value** [Infleqtion/client-superstaq#655]
### What is happening?

See reproduction code below

### How can we reproduce the issue?

```
!pip install qiskit_superstaq qiskit-ibmq-provider

import qiskit
from qiskit import IBMQ
import qiskit_superstaq as qss

qiskit.IBMQ.enable_account(ibm_api_token)
qiskit_provider = qiskit.IBMQ.providers()[1]
ss_provider = qss.SuperstaqProvider(ss_api_token)

backend = qiskit_provider.get_backend("ibmq_jakarta")

qc = qiskit.QuantumCircuit(2, 2)
qc.cx(0, 1)
qc.measure_all()

qc = qiskit.transpile(qc, backend=backend)
qc.draw(fold=-1)
out = ss_provider.ibmq_compile(qc, target="ibmq_jakarta_qpu", dynamical_decoupling=True)
out.circuit.draw()
out.circuit.calibrations
```

<img width="469" alt="image" src="https://github.com/Infleqtion/client-superstaq/assets/3976048/79887ce3-f1ac-42f9-b2d0-ad1c369db176">

### What should happen?

Decomposition is good, but pulse gate circuit has no calibrations, so IBM won't know how to run the acecr

### Environment

latest

**Comment by richrines1** [comment]
ibmq_compile still returns a pulse schedule instead of the pulse gate circuit (existing issue: https://github.com/Infleqtion/server-superstaq/issues/2316, which should now be unblocked)

**Comment by richrines1** [comment]
one weird parity question: in qss, `out.circuit` should clearly just be the pulse gate circuit. In css, `out.circuit` is the compiled cirq circuit - should we put the pulse gate circuit somewhere like `out.qiskit_circuit` / `out.pulse_gate_circuit` / `out.pulse_gate_qiskit_circuit` (and live with the inconsistency of this attribute only existing in css)?

**Comment by paaige** [comment]
@richrines1 @singular-value confirming this can be closed by #730?

**Comment by richrines1** [comment]
yep thanks! i should have linked to this issue
